### DISCUSSION TITLE
Source code for your slides

### DISCUSSION THREAD

**Discussion by jiangshiguo** [rbind/yihui#1477] [Q&A] [answered]
Hi Yihui,

I like the format of your recent presentation: [An anatomy of R Markdown with minimal applications](https://slides.yihui.org/2023-minimal-r-markdown.html). When I downloaded the `2023-minimal-r-markdown.rmd` file, I cannot regenerate your results, likely due to the lack of other files you used, e.g., css files, etc. It is ok if you can share them as well?

Thanks!

Shiguo

**Comment by yihui** [comment, marked as answer]
Hi, you need to use the dev version of **markdown** as I mentioned in the talk: https://slides.yihui.org/2023-minimal-r-markdown.html#9

**Reply by jiangshiguo** [reply]
Thank you!

**Reply by yihui** [reply]
You are welcome! I'm still actively improving this package. I'm glad that you like the presentation format. Please feel free to let me know if you have any suggestions on making this format better (I'm already quite satisfied, but I'd still like to know what other users think).

**Reply by jiangshiguo** [reply]
Thank you for developing this package. I like the feature that the text will automatically reflow based on the size of the screen.
I also checked out the documentation, `markdown/doc/slides.rmd`, which is very useful.

I have one suggestion for the slide navigation: currently, it supports up/down arrows or pageup and pagedown keys. It will be convenient if it also supports the left and right arrows. I am using windows. I don't know if it is different for other systems.

**Reply by yihui** [reply]
You are right about the navigation keys. Left/Right arrows are natural choices for traditional slides. For this presentation format, it is essentially a continuous web page, so you scroll up and down. That said, I can definitely implement your suggestion as a plugin. Thanks!
